config/environments/production.rb

```diff
@@ -41,6 +49,16 @@
   # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
+  # https://guides.rubyonrails.org/active_storage_overview.html
+  # Store uploaded files on the local file system (see config/storage.yml for options).
+  # Disable https://stackoverflow.com/questions/49813214/disable-active-storage-in-rails-5-2
+  # config.active_storage.service = :local
+
+  # Mount Action Cable outside main process or domain.
+  # config.action_cable.mount_path = nil
+  # config.action_cable.url = "wss://example.com/cable"
+  # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
+
```

Comment:
more commented out settings?

Reply:
It's part of what got added when running bin/rails app:update. We're not using but I thought they should be left in.